app/process/roads_bikelanes/bikelanes/BikelanesPresence.lua

```diff
@@ -67,6 +67,11 @@ function BikelanesPresence(object, cycleways)
     presence.left = presence.left or 'not_expected'
   end
 
+  -- Set left side to "not_expected" for junctions
```

Comment:
Alternativ über implicit oneway lösen oder sogar einen OSM-Tag "forcen" auf object.tags in einem Pre-Processing Step.